app/controllers/api/v1/status_phone_controller.rb

```diff
@@ -7,7 +7,16 @@ def create
         return wrong_phone unless store
         return forbidden_store unless current_user.stores.pluck(:id)&.include?(store.id)
 
-        StatusStoreOwner.create!(status: params['status'],
+        status = case params['status']
+                 when '1'
+                   0
+                 when '2'
+                   5
+                 when 3
```

Comment:
Why is this a number and the other two are strings?

Reply:
Cause I'm an idiot, I'll replace it.